src/deploy/functions/runtimes/discovery/v1alpha1.ts

```diff
@@ -243,6 +243,24 @@ function parseEndpoints(
       "ingressSettings",
       "environmentVariables"
     );
+    renameIfPresent(
+      parsed,
+      ep,
+      "secretEnvironmentVariables",
+      "secretEnvironmentVariables",
+      (senvs: ManifestEndpoint["secretEnvironmentVariables"]) => {
+        if (senvs && senvs.length > 0) {
+          ep.secretEnvironmentVariables = [];
+          for (const { key, secret } of senvs) {
+            ep.secretEnvironmentVariables.push({
+              key: key,
+              secret: secret || key, // if secret is undefined, assume env var key == secret name
+              projectId: project,
```

Comment:
2 Nits (feel free to ignore)

* rename `project` to `projectId` (its more descriptive)
* if you rename `project` to `projectId`, then you can also do shorthand here

Reply:
I'm going to prefer to keep `project` here since that's the name of the property we use to store project identifiers in `backend.Endpoint`. Is that a good idea? I dunno, but that change would touch LOTS of files and I'd suggest that we do that in a separate PR if we want to make that change.